main.js

```diff
@@ -18,6 +21,27 @@ var bindings = require('./bindings'),
     requireJsGraph = require('./requireJsGraph');
     ArgumentParser = require('argparse').ArgumentParser;
 
+
+const findJsFilesInDir = function(dir, filelist) {
```

Comment:
additional command line argument - you can pass a directory and it will include all files in that directory when producing a graph